Description: Shrink the height of GNOME Shell's message tray
 The message tray takes too much screen space at 72 px, and most status icons
 seem to be half that height, so make it 36 px instead.
Author: Vincent Cheng <Vincentc1208@gmail.com>
Last-Update: 2012-08-28

--- a/Zukitwo-Shell/gnome-shell/gnome-shell.css
+++ b/Zukitwo-Shell/gnome-shell/gnome-shell.css
@@ -1429,11 +1429,11 @@
 	box-shadow: inset 0 0 3px 0 rgba(0,0,0,0.5);
 	border: 0px solid rgba(255,255,255,0.05);
 	border-top-width: 1px;
-	height: 72px;
+	height: 36px;
 }
 
 .message-tray-summary {
-	height: 72px;
+	height: 36px;
 }
 
 .no-messages-label {
